datahub-classify/src/datahub_classify/infotype_utils.py

```diff
@@ -82,15 +92,30 @@ def detect_named_entity_spacy(
 def perform_basic_checks(
     metadata: Metadata,
     values: List[Any],
-    config_dict: Dict[str, Dict],
+    config_dict: Dict[str, Union[Dict, List[str], None]],
     infotype: str,
     minimum_values_threshold: int,
 ) -> bool:
     basic_checks_status = True
+    metadata.name = (
+        metadata.name
+        if not config_dict.get("strip_formatting")
+        else strip_formatting(metadata.name)
+    )
+    prediction_factors = config_dict.get(PREDICTION_FACTORS_AND_WEIGHTS)
+    exclude_name = config_dict.get(EXCLUDE_NAME, [])
     if (
-        config_dict[PREDICTION_FACTORS_AND_WEIGHTS].get(VALUES, None)
+        isinstance(prediction_factors, dict)
+        and prediction_factors.get(VALUES, None)
         and len(values) < minimum_values_threshold
     ):
+        logger.warning(
+            f"The number of values for column {metadata.name}"
+            f"does not meet minimum threshold for {infotype}"
+        )
+        basic_checks_status = False
+    elif exclude_name is not None and metadata.name in exclude_name:
+        logger.warning(f"Excluding match for {infotype} on column {metadata.name}")
```

Comment:
same for this. As this is executed for every column of every table, the real ingestion logs become pretty difficult to debug.

If I remember correctly, there is already an aggregated table level warning logs statement if the columns were skipped due to basic  It does not display the reason details, but that should be good enough to get first indication.

Reply:
done in this PR: https://github.com/acryldata/datahub-classify/pull/21